src/vue/pages/Issuance.vue

```diff
@@ -0,0 +1,204 @@
+      <upload-pre-issuance-form :is-shown.sync="isPreIssuanceDrawerShown" />
+    </drawer>
+    <drawer :is-shown.sync="isCreateDrawerShown">
+      <template slot="heading">
+        {{ 'issuance.create-issuance' | globalize }}
+      </template>
+      <create-issuance-form :is-shown.sync="isCreateDrawerShown" />
+    </drawer>
+    <div class="issuance" v-if="isLoaded">
+      <div class="issuance-history__table">
+        <table class="app__table">
+          <thead>
+            <tr>
+              <th>
+                {{ 'issuance.counterparty' | globalize }}
+              </th>
+              <th>
+                {{ 'issuance.amount' | globalize }}
+              </th>
+              <th>
+                {{ 'issuance.asset-code' | globalize }}
+              </th>
+              <th>
+                {{ 'issuance.date' | globalize }}
+              </th>
+              <th>
+                {{ 'issuance.reference' | globalize }}
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr v-for="issuance in issuanceHistory" :key="issuance.id">
+              <td v-if="issuance.counterparty">
+                {{ issuance.counterparty }}
+              </td>
+              <td v-else>
+                <loader
+                  :message-id="'issuance.lbl-loading'"
+                />
+              </td>
+              <td>
+                {{
+                  issuance.amount | formatMoney
+                }}
+              </td>
+              <td>
+                {{
+                  issuance.asset
+                }}
+              </td>
+              <td>
+                {{ issuance.ledgerCloseTime | formatCalendar }}
+              </td>
+              <td>
+                {{ issuance.reference }}
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+    <div v-else-if="!isLoadingFailed">
+      <loader
+        :message-id="'issuance.lbl-loading'"
+      />
+    </div>
+    <div v-else>
+      <p>
+        {{ 'issuance.lbl-loading-error' | globalize }}
+      </p>
+    </div>
+  </div>
+</template>
+
+<script>
+import Loader from '@/vue/common/Loader'
+import Drawer from '@/vue/common/Drawer'
+import TopBar from '@/vue/common/TopBar'
+import CreateIssuanceForm from '@/vue/forms/CreateIssuanceForm'
+import UploadPreIssuanceForm from '@/vue/forms/UploadPreIssuanceForm'
+
+import { Sdk } from '@/sdk'
+// FIXME: move XDR-dependent object imports to sdk
+import { ACCOUNT_TYPES, OPERATION_TYPES } from '@/js/const/xdr.const'
+
+import { mapGetters } from 'vuex'
+import { vuexTypes } from '@/vuex'
+
+import { errors } from '@tokend/js-sdk'
+
+const PAGE_LIMIT = 200
+
+export default {
+  name: 'issuance',
+  components: {
+    Loader,
+    Drawer,
+    TopBar,
+    CreateIssuanceForm,
+    UploadPreIssuanceForm
+  },
+  data: _ => ({
+    issuanceHistory: null,
+    isLoaded: false,
+    isLoadingFailed: false,
+    isCreateDrawerShown: false,
```

Comment:
`Create` what?

and btw the phrase `create issuance` looks awkward for me. I think we can just use `issuance` instead everywhere

Reply:
@quinn71 you restricted just the same thing in of of the prev reviews with `isSidebarShown` in `Sidebar.vue`. here's just the same issue `isCreateDrawerShown` in the `Issuance.vue`. Definitely, `isCreateDrawerShown` here means create issuance drawer shown

Reply:
Resolved, renamed